src/lib/icons/OpenChat.svelte

```diff
@@ -0,0 +1,43 @@
+<svg viewBox="0 0 364 364" xmlns="http://www.w3.org/2000/svg">
+	<defs>
+		<linearGradient id="gradient-2" />
+		<linearGradient id="gradient-5">
+			<stop style="stop-color: rgb(64, 64, 64);" offset="0" />
+			<stop style="stop-color: rgb(128, 128, 128);" offset="1" />
+		</linearGradient>
+		<linearGradient id="gradient-6">
+			<stop style="stop-color: rgb(128, 128, 128);" offset="0" />
+			<stop style="stop-color: rgb(255, 255, 255);" offset="1" />
+		</linearGradient>
+		<linearGradient
+			id="gradient-5-1"
+			gradientUnits="userSpaceOnUse"
+			x1="974.005"
+			y1="99.877"
+			x2="974.005"
+			y2="388.865"
+			gradientTransform="matrix(-0.999851, -0.016909, 0.017656, -1.110698, 1932.460787, 509.602999)"
+			xlinkHref="#gradient-5"
+		/>
+		<linearGradient
+			id="gradient-6-0"
+			gradientUnits="userSpaceOnUse"
+			x1="188.919"
+			y1="1.638"
+			x2="188.919"
+			y2="361.638"
+			gradientTransform="matrix(-0.465435, -0.885082, 1.172947, -0.616813, 14.606062, 499.208888)"
+			xlinkHref="#gradient-6"
+		/>
+	</defs>
+	<path
+		d="M 188.919 181.638 m -180 0 a 180 180 0 1 0 360 0 a 180 180 0 1 0 -360 0 Z M 188.919 181.638 m -100 0 a 100 100 0 0 1 200 0 a 100 100 0 0 1 -200 0 Z"
+		style="fill: url(#gradient-6-0);"
+		transform="matrix(-0.701584, 0.712586, -0.712586, -0.701584, 442.975678, 174.812952)"
+	/>
+	<path
+		style="stroke-width: 0px; paint-order: stroke; fill: url(#gradient-5-1);"
+		transform="matrix(1.031731, 0.000001, 0, 1.020801, -703.597412, -67.455116)"
+		d="M 958.327234958 100.664699414 A 175.433 175.433 0 0 1 958.327234958 388.077300586 L 913.296322517 323.766492741 A 96.924 96.924 0 0 0 913.296322517 164.975507259 Z"
+	/>
+</svg>
```

Comment:
<svg height="30px" width="30px" viewBox="0 0 364 364" xmlns="http://www.w3.org/2000/svg">
	<defs>
		<linearGradient
			id="gradient-5"
			gradientUnits="userSpaceOnUse"
			x1="0%"
			y1="0%"
			x2="100%"
			y2="100%"
		>
			<stop offset="0%" style="stop-color: var(--svg-fill-color);" />
			<stop offset="100%" style="stop-color:  var(--svg-fill-color);" />
		</linearGradient>
		<linearGradient
			id="gradient-6"
			gradientUnits="userSpaceOnUse"
			x1="0%"
			y1="0%"
			x2="100%"
			y2="100%"
		>
			<stop offset="0%" style="stop-color: var(--svg-fill-color);" />
			<stop offset="100%" style="stop-color: var(--svg-opposite-color);" />
		</linearGradient>
	</defs>
	<path
		d="M 188.919 181.638 m -180 0 a 180 180 0 1 0 360 0 a 180 180 0 1 0 -360 0 Z M 188.919 181.638 m -100 0 a 100 100 0 0 1 200 0 a 100 100 0 0 1 -200 0 Z"
		style="fill: url(#gradient-6);"
		transform="matrix(-0.701584, 0.712586, -0.712586, -0.701584, 442.975678, 174.812952)"
	/>
	<path
		style="stroke-width: 0px; paint-order: stroke; fill: url(#gradient-5);"
		transform="matrix(1.031731, 0.000001, 0, 1.020801, -703.597412, -67.455116)"
		d="M 958.327234958 100.664699414 A 175.433 175.433 0 0 1 958.327234958 388.077300586 L 913.296322517 323.766492741 A 96.924 96.924 0 0 0 913.296322517 164.975507259 Z"
	/>
</svg>

Reply:
The open chat logo seems a bit off. From this code you can set nice colors to make it nicer + you avoid some unnecessary gradient-6-0 and 5-1